test/library/gui/dwarfmode.lua

```diff
@@ -64,4 +64,12 @@ function test.enterSidebarMode()
     guidm.enterSidebarMode(df.ui_sidebar_mode.Default)
     expect.eq(df.ui_sidebar_mode.Default, df.global.ui.main.mode)
     expect.eq('dwarfmode/Default', dfhack.gui.getCurFocus(true))
+
+    -- verify that all supported modes lead where we say they'll go
+    for k,v in pairs(guidm.SIDEBAR_MODE_KEYS) do
+        guidm.enterSidebarMode(k)
+        expect.eq(k, df.global.ui.main.mode)
```

Comment:
FYI, this is a situation where I'd use the `comment` argument to provide information about which iteration of the loop failed. It's fairly easy to determine if non-equal numbers get printed in this case, though. I dealt with a couple similar cases here: 1014c41d

Reply:
Agree. Thanks for adding